src/main/java/com/github/dockerjava/netty/exec/RenameContainerCmdExec.java

```diff
@@ -0,0 +1,31 @@
+package com.github.dockerjava.netty.exec;
+
+import com.github.dockerjava.api.command.RenameContainerCmd;
+import com.github.dockerjava.core.DockerClientConfig;
+import com.github.dockerjava.netty.MediaType;
+import com.github.dockerjava.netty.WebTarget;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * rename a container.
+ */
+public class RenameContainerCmdExec extends AbstrSyncDockerCmdExec<RenameContainerCmd, Void> implements RenameContainerCmd.Exec {
```

Comment:
Could you try wrote javadoc in interface and check that it inherited by default to classes? (in idea just check javadoc on this class)